### ISSUE TITLE
Include instructions on auto-rebuilding on save?

### ISSUE THREAD

**Issue by oxinabox** [piever/Remark.jl#37]
I have auto-rebuilding for everytime i save using the `FileWatching` standardlib.
Which is nice to keep running one julia process becuase time to first plot.

My script right now looks like:

```julia
using Pkg: Pkg
cd(@__DIR__)
Pkg.activate(pwd())
using Remark: Remark

function build()
    dir = Remark.slideshow(
        "content_src/pres.md", "out";
        css="content_src/style.css"    
    )
    cp("content_src/pictures", joinpath(dir, "build", "pictures"); force=true)
    return dir
end

using FileWatching
path = "src/pres.md"
while true
    build()
    @info "Rebuilt"
    FileWatching.watch_folder("content_src/")
end
```

**Comment by piever** [comment]
Yes, this should definitely be mentioned (I confess I didn't know it was so easy to get it to work).

The only thing that I'm slightly unhappy about is the need to manually copy a directory, I think that should be done automatically (see https://github.com/piever/Remark.jl/issues/22). Maybe the rule could be that if there is a folder named `assets` in the same folder as the main file, then it is copied in the `build` directory next to `index.html`, so that local paths keep on working. Does that make sense?

**Comment by oxinabox** [comment]
Probably, it isn't the point of this issue.

**Comment by oxinabox** [comment]
#42

**Comment by piever** [comment]
Fixed by #42
